index.html

```diff
@@ -3,173 +3,156 @@
+	<script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.0/dist/umd/popper.min.js" integrity="sha384-Q6E9RHvbIyZFJoft+2mJbHaEWldlvI9IOYy5n3zV9zzTtmI3UksdQRVvoxMfooAo" crossorigin="anonymous"></script>
+	<script src="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js" integrity="sha384-wfSDF2E50Y2D1uUdj0O3uMBJnjuUD4Ih7YwaYd1iqfktj0Uod8GCExl3Og8ifwB6" crossorigin="anonymous"></script>
+	<meta name="author" content="Ayush Shridhar">
 	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<script src="https://ajax.googleapis.com/ajax/libs/jquery/2.2.4/jquery.min.js"></script>
+	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+	<link rel="stylesheet" href="./assets/css/styles.css">
 </head>
+		<script>
+		$(window).load(function() {
+          $(".se-pre-con").fadeOut("slow");;
+        });
+		</script>
 	<body>
-		<!-- Navbar -->
-		<nav class="nav container void-background">
-			<!-- This "nav-menu" is hidden on mobile -->
-			<!-- Add the modifier "is-active" to display it on mobile -->
-			<div class="nav-left">
-				<a href="http://medium.com/@ayush1999/" class="nav-item">
-					<span class="icon">
-						<i class="fa fa-medium"></i>
-					</span>
-				</a>
-				<a href="http://github.com/ayush1999" class="nav-item">
-					<span class="icon">
-						<i class="fa fa-github"></i>
-					</span>
-				</a>
-				<a href="https://twitter.com/ayuSHridhar" class="nav-item">
-					<span class="icon">
-						<i class="fa fa-twitter"></i>
-					</span>
-				</a>
+		<!-- PRE LOADER  -->
+		<div class="se-pre-con"></div>
+		<!-- NAVBAR -->
+		<div class="container">
+		<nav class="navbar navbar-expand-sm">
+			<div class="move">
+				<a href ="https://medium.com/@ayush1999/" class ="fa fa-medium nav"></a>
+				<a href ='https://github.com/ayush-1506' class ="fa fa-github nav"></a>
+				<a href = 'https://twitter.com/ayuSHridhar' class ="fa fa-twitter nav"></a>
 			</div>
-
-			<div class="nav-right nav-menu">
-				<a class="nav-item" href="#about">About</a>
-				<a class="nav-item" href="#projects">Projects</a>
-				<a class="nav-item" href="#social">Social</a>
+			<button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+				<span class="navbar-toggler-icon">   
+					<i class="fa fa-navicon" style="color:#fff; font-size:28px;"></i>
+				</span>
+			</button>
+		  
+			<div class="collapse navbar-collapse" id="navbarSupportedContent">
+			  <ul class="navbar-nav ml-auto">
+				<li class="nav-item active">
+				  <a class="nav-link cus" href="#about">About</a>
+				</li>
+				<li class="nav-item">
+				  <a class="nav-link cus" href="/blog/main.html">Blog</a>
```

Comment:
Test changes locally before committing. (This href doesn't exist. I believe this should be blog/index.html).